### PR TITLE
Add screenshots and text for all Advanced browser options

### LINKED ISSUE #1726: [Request] Add info on all Elastic Synthetics "Advanced Browser options"
Follow up from #1720

## Description

>>I added information on Throttling options as requested in https://github.com/elastic/observability-docs/issues/1411, but there's no mention of other Advanced Browser options. Should those also be added?
>
>Good point, I hadn’t actually realised they weren’t documented (and so through the throttling addition was going into existing docs). It would be useful to include these - could you do this in a follow up issue/PR?

We should add information on other Advanced Browser options in the Elastic Synthetics integration.

## Collaboration

The docs team will lead producing the content

**Contact Person:** @paulb-elastic 

## Suggested Target Release

@paulb-elastic in which release was this introduced?

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Fixes #1726 

Adds information on "Selective tests" and "Synthetics agent options" to [Synthetic monitoring using Elastic Agent and Fleet](https://www.elastic.co/guide/en/observability/master/synthetics-quickstart-fleet.html).

### CODE REVIEW

File: docs/en/observability/synthetics-quickstart-fleet.asciidoc [hunk 1]
```diff
@@ -71,8 +71,22 @@ remote zip endpoint for the git repository for https://github.com/elastic/synthe
 Using a git repository for synthetics journeys allows you to use a GitOps workflow for deploying changes to synthetic monitors,
 enabling git features, like pull requests and reverts, for managing tests more effectively.
 
-Click *Advanced Browser options* to see more ways to configure your monitor including _Throttling options_.
-Set throttling parameters to more accurately control the network speed that the monitor represents.
+Click *Advanced Browser options* to see more ways to configure your monitor.
+
+Use the *Selective tests* options to only run a subset of all the tests in your suite.
```
Comment (dedemorton) [inline docs/en/observability/synthetics-quickstart-fleet.asciidoc:76]:
```suggestion
Use the *Selective tests* options to run a subset of all the tests in your suite.
```

File: docs/en/observability/synthetics-quickstart-fleet.asciidoc [hunk 2]
```diff
@@ -71,8 +71,22 @@ remote zip endpoint for the git repository for https://github.com/elastic/synthe
...
+Only the configured subset will be run by this monitor.
+These options are only available when using the _Zip URL_ source type.
+
+[role="screenshot"]
+image::images/synthetics-integration-selective-tests.png[Use the Selective tests options to only run a subset of all the tests in your suite in Elastic {fleet}]
```
Comment (dedemorton) [inline docs/en/observability/synthetics-quickstart-fleet.asciidoc:81]:
```suggestion
image::images/synthetics-integration-selective-tests.png[Use the Selective tests options to run a subset of all the tests in your suite in Elastic {fleet}]
```

File: docs/en/observability/synthetics-quickstart-fleet.asciidoc (hunk 1 above)
Comment (dedemorton) [reply docs/en/observability/synthetics-quickstart-fleet.asciidoc:76]: I don't think "only" is really needed here since you say subset.

**Review by dedemorton — COMMENTED** [review]
Nice work! Just a few minor suggestions.

**Review by paulb-elastic — APPROVED** [review]
Thanks for the updates - LGTM

**Review by dedemorton — APPROVED** [review]
LGTM. Nice work!

**Pull request merged**